eli5/formatters/image.py

```diff
@@ -97,8 +98,9 @@ def format_as_image(expl, # type: Explanation
         raise TypeError('Explanation image must be a PIL.Image.Image instance. '
                         'Got: {}'.format(image))
     if image.mode != 'RGBA':
-        raise ValueError('Explanation image must have mode "RGBA". '
-                         'Got image with mode: %s' % image.mode)
+        # normalize to 'RGBA'
+        warnings.warn('Converting image to RGBA.', stacklevel=2)
```

Comment:
Why do we want to warn about this? I think usually warnings are used to warn about something that is suboptimal and likely needs to be fixed, but I think in this case it's completely fine to pass any image?

Reply:
As discussed the two warnings may help the user/developer in case of unexpected behaviour, i.e. image implementation is dispatched but wanted something else, or image is converted to RGBA but the resulting image doesn't look right). However, it's better to replace them with logging (possibly in a separate PR to add logging to the library).